### PR TITLE
Adding terraform provider packages

### PR DESCRIPTION
**Pull request by emesika**
This patch adds the terraform provider go packages needed for terraform
init command as plugin sources.

Signed-off-by: emesika <emesika@redhat.com>

## Please describe the change you are making

...

## Are you the owner of the code you are sending in, or do you have permission of the owner?

...

## The code will be published under the BSD 3 clause license. Have you read and understood this license?

...

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: .github/workflows/compile-ocp-installer.yml [hunk 1]
```diff
@@ -14,11 +14,14 @@ jobs:
       - uses: actions/checkout@v2
       - name: prepare env
         run: |
-           mkdir -p ${GITHUB_WORKSPACE}/BUILDROOT  ${GITHUB_WORKSPACE}/SOURCES
-           yum install -y git createrepo_c rpm-build golang
+           mkdir -p ${GITHUB_WORKSPACE}/BUILDROOT  ${GITHUB_WORKSPACE}/SOURCES ${GOPATH}
```
Comment (janosdebugs) [inline .github/workflows/compile-ocp-installer.yml:17]: Where is `$GOPATH` set? Is this

**Review by janosdebugs — APPROVED** [review]
Looks good, a bit confused about the `$GOPATH` variable.

**Comment by janosdebugs** [comment]
ci test please

**Comment by janosdebugs** [comment]
ci add to whitelist

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Review by janosdebugs — APPROVED** [review]

**Pull request merged**